src/lib.rs

```diff
@@ -98,6 +98,33 @@ impl Client {
         self.drive_receive(parse_ascii_response).await
     }
 
+    pub(crate) async fn get_write_multi_responses(
```

Comment:
This method is oddly named imo. `get` is the memcached command. `set_multi` should be used instead of `write_multi`.

Maybe this is more like `filter_set_multi_responses`?